### PR TITLE
Upload file

### PR DESCRIPTION
**Pull request by Maxnm47**

### CODE REVIEW

File: src/public/client.js [hunk 1]
```diff
@@ -72,3 +72,37 @@ rdyButton = document.querySelector('#connecterBtn');
...
+    // gets the file
+    const file = uploadFileEle.files[0];
+    const reader = new FileReader();
+    reader.onload = (event) => {
+      const weights2 = JSON.parse(event.target.result); // parse JSON data
+      // Send weights to server using fetch() with a POST request
+      fetch('/server-weights', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json', // set the Content-Type header
```
Comment (Tinggaard) [inline src/public/client.js:89]: Massivt god kommentar.

**Review by enevoldsen1oo — APPROVED** [review]
Super nice working very fine :)

**Review by enevoldsen1oo — CHANGES_REQUESTED** [review]
Please make it work for me also :)

**Review by Tinggaard — CHANGES_REQUESTED** [review]
Lav en indikation på at filen er uploadet. Og lav CSS pænt.

File: src/public/client.js [hunk 2]
```diff
@@ -72,3 +72,63 @@ rdyButton = document.querySelector('#connecterBtn');
...
+  fileChecker(fileInputElement);
+  if (correctInput) {
+    fileUpdate(fileInputElement);
+  }
+  correctInput = false; // reset the stuff :)
+});
+// const fileBtn = document.querySelector('#fileSendButton');
+// fileBtn.addEventListener('click', () => {
+//   fileSender();
+// });
```
Comment (Tinggaard) [inline src/public/client.js:134]: nani?!?

File: src/public/index.html [hunk 3]
```diff
@@ -9,8 +9,17 @@
       <div class="homeButton"><button>Home</button></div>
       <div class="statsButton"><button>Stats</button></div>
       <div class="userButton"><button>User</button></div>
+      <div class="fileInputDiv">
+        <label for="fileInput" class="uploadButton" id="fileInputLabel">Select file</label>
+        <input class="fileInput hidden" type="file" id="fileInput" />
+        <!-- <input type="button" value="Upload" class="fileSendButtonClass" id="fileSendButton" /> -->
```
Comment (Tinggaard) [inline src/public/index.html:15]: why is dis here?

**Review by Tinggaard — CHANGES_REQUESTED** [review]

File: src/public/index.html (hunk 3 above)
Comment (Maxnm47) [reply src/public/index.html:15]: Fordi jeg elsker mænd

File: src/public/client.js (hunk 2 above)
Comment (Maxnm47) [reply src/public/client.js]: fordi jeg er semi-bisexual, jeg kan kun lide mænd

**Review by Tinggaard — APPROVED** [review]

**Review by enevoldsen1oo — APPROVED** [review]
Yes, I, Martin hereby approve these changes. For I like them

**Pull request merged**